.github/workflows/audit.yml

```diff
@@ -3,21 +3,40 @@
 name: Audit
 
 on:
+  workflow_dispatch: null
   schedule:
     # "At 01:00 on Monday" https://crontab.guru/#0_1_*_*_1
     - cron: "0 1 * * 1"
-  workflow_dispatch:
 
 jobs:
   audit:
-    name: npm audit
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
```

Comment:
Is all this cause of templating under the hood or did you manually add this? Our linting workflow runs under ubuntu and doesn't need all this windows stuff.

Reply:
Yes, all of the yml for node setup is in a single template that gets included everywhere. It would be possible to add logic to only include what we need in each generated file, but given that they are generated I opted to only use the yml if statements to conditionalize the behavior of each action instead of changing what is actually generated.

I'm not sold on either solution, so we should decide on where we want to offload this complexity.

Reply:
Just making sure. I'm fine if it's just How npm Is Installed, just wanted to be sure it was because of that and not some accidental manual copying.